src/main/java/model/car/CarRepository.java

```diff
@@ -0,0 +1,20 @@
+package model.car;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CarRepository {
+    private final List<Car> cars = new ArrayList<>();
+
+    public CarRepository() {
+    }
+
+    public List<Car> cars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public void addCars(Car car) {
```

Comment:
addCar 가 더 적절하지 않을까요?
```suggestion
    public void addCar(Car car) {
```